core_lib/session/user_security.py

```diff
@@ -35,9 +37,15 @@ def token_to_session_object(self, token):
             self.logger.error(ex)
             return None
 
-    def _secure_entry(self, request, policies):
+    def _secure_entry(self, request, policies, frame_work_type='django'):

```

Comment:
rather than passing it as a parameter check if we can do this instead,

when initializing your Flask app, add this 
```python
from core_lib.web_helpers.web_helprs_utils import WebHelpersUtils

WebHelpersUtils.init(web_util.ServerType.FLASK)
```

and in the function, you can do this
```python
from core_lib.web_helpers.web_helprs_utils import WebHelpersUtils

if WebHelpersUtils.get_server_type() == WebHelpersUtils.ServerType.FLASK
```